rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -11,17 +11,24 @@ if [ -z `getent passwd spinnaker` ]; then
   useradd --gid spinnaker spinnaker -m --home-dir /home/spinnaker
 fi
 
+create_temp_dir() {
+  PATH=$PATH:/usr/local/bin
+  TEMPDIR=$(mktemp -d /tmp/installrosco.XXXX)
```

Comment:
Added /tmp/ prefix otherwise it was creating the temp directory in the root / directory.